modules/ROOT/pages/partner-manager-architecture.adoc

```diff
@@ -1,2 +1,63 @@
+
+When you deploy a configured message flow, Partner Manager automatically translates the configurations used in the message flow and deploys and updates a set of applications necessary to orchestrate end-to-end B2B processing to the Mule runtime. These applications include:
+
+* Receive endpoint services
+* Document processing service
+* Replication service
+* Send endpoint services
+
+image::partner-manager-high-level-arch.png[Anypoint Partner Manager Architecture]
+
+[%header%autowidth.spread]
+|===
+|Callout |Name |Description
+|image:pm-arch-diagram-1.png[]
+|Partner Manager
+|MuleSoft hosted cloud control plane allowing users to configure trading partner profiles, build and deploy B2B integration message flows, and monitor B2B transactions activity.
+
+|image:pm-arch-diagram-2.png[]
+|Receive Endpoint Services
+|Service to receive inbound B2B or EDI messages from partners transport protocols such as AS2 and SFTP or outbound application messages from your organization’s backend systems via HTTP, HTTPS, or SFTP.
+
+|image:pm-arch-diagram-3.png[]
+|Document Processing Service
+|Core B2B processing engine responsible for parsing, validating and translating received B2B messages based on configurations and mapping in Partner Manager.
+
+|image:pm-arch-diagram-4.png[]
+|API Replication Service
+|Tracking service responsible for updating transmissions and transaction activities to the control plane and to the payload storage API.
+
+|image:pm-arch-diagram-5.png[]
+|Send Endpoint Services
+|Service to deliver translated inbound application messages to internal applications such as ERP, billing, Order Management via HTTP, HTTPS or SFTP; or outbound B2B/EDI payloads to your trading partners via AS2, HTTP, HTTPS or SFTP.
+
+|image:pm-arch-diagram-6.png[]
+|Payload Storage API
+|API to store B2B payload data in your own storage solution.
+|===
+
+== Partner Manager for CloudHub
+
+When deploying B2B Integration message flows to CloudHub, Anypoint MQ queues are used between processing stages to add additional reliability to the B2B message processing.
+
+As of the time of first deployment from Partner Manager, runtime applications are deployed by default to the CloudHub region specified in the business group’s default settings in Runtime Manager. Anypoint MQ queues are automatically created during the deployment process in the same or the nearest available region. Any new runtime application from subsequent deployments is deployed to the same region as the first deployment.
+
+image::partner-manager-cloud-deployment.png[Anypoint Partner Manager for CloudHub]
+
+== Partner Manager for On-Premise Mule Runtime
+
+When deploying B2B Integration message flows to your own hosted on-premise Mule runtime, HTTP calls are used between processing stages during B2B message processing.
+
+The runtime applications are deployed to the server group (B2BServerGroup) that you will have configured in your data center or your private cloud.
```

Comment:
passive voice > active

Reply:
"will" > present tense

Reply:
I am currently checking in with Fer, Hanna, and Duke about the subtleties around using "runtime," which have never been crystal clear to me. If referring to the engine, I know we want to use one of Mule runtime engine (Mule) or refer to the on-prem "instance." 
I'll share what I find out; please feel free to correct/share with me!

Reply:
P.S. Don't let this slow you down.

Reply:
Done!